### ISSUE TITLE
[Accessibility : Keyboard & navigation] Add Adapter/ECKD/FCP Device page not getting closed using ESC button

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/gingers390x#48]
Add bond/vlan modal window should be closed using ESC button click. User should not be trapped in the window.
